build.sbt

```diff
@@ -0,0 +1,96 @@
+import Benchmark._ // see project/Benchmark.scala
+import Dependencies._ // see project/Dependencies.scala
+
+val buildVersion = "0.10.1-SNAPSHOT"
+
+def commonSettings = Seq(
+  version in ThisBuild := buildVersion,
+  scalaVersion := Util.buildScalaVersion,
+  organization in ThisBuild := "org.scala-lang.modules",
+  organizationName in ThisBuild := "LAMP/EPFL",
+  organizationHomepage in ThisBuild := Some(url("http://lamp.epfl.ch")),
+  homepage in ThisBuild := Some(url("https://github.com/scala/pickling")),
+  licenses in ThisBuild := List("BSD-like" -> url("http://www.scala-lang.org/downloads/license.html")),
+  scmInfo in ThisBuild := Some(ScmInfo(url("https://github.com/scala/pickling"), "git@github.com:scala/pickling.git")),
+  developers in ThisBuild := List(
+    Developer("xeno-by", "Eugene Burmako", "@xeno_by", url("http://github.com/xeno-by")),
+    Developer("heathermiller", "Heather Miller", "@heathercmiller", url("http://github.com/heathermiller")),
+    Developer("phaller", "Philipp Haller", "@philippkhaller", url("http://github.com/phaller")),
+    Developer("havocp", "Havoc Pennington", "@havocp", url("https://github.com/havocp")),
+    Developer("eed3si9n", "Eugene Yokota", "@eed3si9n", url("https://github.com/eed3si9n")),
+    Developer("jsuereth", "Josh Suereth", "@jsuereth", url("https://github.com/jsuereth"))
+  ),
+  resolvers += Resolver.sonatypeRepo("snapshots"),
+  resolvers += Resolver.sonatypeRepo("releases"),
+  scalacOptions ++= Seq("-feature"),
+  parallelExecution in Test := false, // hello, reflection sync!!
+  conflictWarning in ThisBuild := ConflictWarning.disable,
```

Comment:
That's copy-pasted from old build.scala. I think it was a workaround for #142 (sbt scala version warning), which should no longer be needed since organization is not org.scala-lang any more.